rpcd/playbooks/roles/filebeat/defaults/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+filebeat_apt_gpg_keys:
+  # NOTE(sigmavirus24): Yes, it really is this key:
+  # https://www.elastic.co/guide/en/beats/libbeat/current/setup-repositories.html
+  - { url: "https://packages.elasticsearch.org/GPG-KEY-elasticsearch", state: present }
+
+filebeat_apt_repo_url: "http://packages.elastic.co/beats/apt"
+
+filebeat_apt_repos:
+  - { repo: "deb {{ filebeat_apt_repo_url }} stable main", state: present }
+
+filebeat_apt_packages:
+  - "filebeat"
+
+filebeat_logstash_port: "{{ logstash_beats_port |default(5044) }}"
+
+# NOTE(sigmavirus24): Once we upgrade to Ansible 2.1, we can use the extract
+# filter
+#"{{ groups['logstash_all'] | map('extract', hostvars, ['container_address']) | list }}"
+filebeat_logstash_hosts: "{% for host in groups['logstash_all'] %}{{ hostvars[host]['ansible_ssh_host'] }}:{{ filebeat_logstash_port }}{% if not loop.last %},{% endif %}{% endfor %}"
+
+filebeat_config_overrides:
+  filebeat:
+    prospectors:
+      - paths:
+        - /var/log/*.log
+        - /var/log/*/*.log
+        encoding: utf-8
+        input_type: log
+    registry_file: /var/lib/filebeat/registry
+  output:
+    logstash:
+      hosts: "{{ filebeat_logstash_hosts.split(',')|list }}"
```

Comment:
Why not directly writing the logstash hosts as a list, instead of writing "," and later splitting "," for doing a list?